src/lib/VTX/devVTX.cpp

```diff
@@ -25,6 +25,22 @@ void VtxTriggerSend()
     devicesTriggerEvent();
 }
 
+void VtxPitmodeSwitchUpdate()
+{
+  if (config.GetVtxPitmode() > 1)
```

Comment:
This would be nicer as a guard, but it's minor, so feel free to ignore.
i.e:
`if (config.GetVtxPitmode() == 0) { // pitmode is off
    return;
}`